src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayAmount = 40;
+  let totalAmount = 0;
```

Comment:
You don't need this variable, just return the result of the calculation each  time